instance/server/Shared/Routes.elm

```diff
@@ -49,18 +48,29 @@ type alias Assets =
     , noredinkLogo : Image
     }
 
-
+routes : List (Matcher Route)
 routes =
-    { apply = "/apply"
-    , index = "/"
-    , signup = "/signup"
-    , startTest = "/start-test"
-    , login = "/login"
-    , registerUser = "/admin/registerUser"
-    , swimlanes = "/swim"
-    , viewSingleUser = "/admin/viewUser"
-    }
+    [ static Index "/"
+    , static Apply "/apply"
+    , static SignUp "/signup"
+    , static StartTest "/start-test"
+    , static Login "/login"
+    , static RegisterUser "/admin/registerUser"
+    , static Swimlanes "/swim"
+    , static ViewSingleUser "/admin/viewUser"
```

Comment:
not sure about the string parser approach for get arguments, it's a bit more complicated as order should not matter.